src/vs/workbench/parts/terminal/electron-browser/media/terminal.css

```diff
@@ -29,6 +29,7 @@
 	position: absolute;
 	bottom: 0;
 	top: 0;
+	padding-bottom: 2px;
```

Comment:
I tried quite a bit of combinations of both margin and padding as well as trying to change the positioning.
I was unable to use just position-bottom, this threw everything off.
even if we used bottom I believe we would still have to do the inner height adjustment so I'm not sure if the change is if I'm understanding you.